website/versioned_docs/version-0.19.0/tutorial/index.mdx

```diff
@@ -478,6 +478,13 @@ serde = { version = "1.0", features = ["derive"] }
 wasm-bindgen-futures = "0.4"
 ```
 
+Import `Deserialize` and `Request`:
+```rust ,ignore {2-3}
+use yew::prelude::*;
++ use serde::Deserialize;
++ use reqwasm::http::Request;
+```
+
```

Comment:
I would move these to the next snippet and remove the "Import `Deserialize` and `Request`" line